x11-wm/compiz/compiz-0.8.14.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils gnome2-utils
+
+DESCRIPTION="OpenGL window and compositing manager"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+ LGPL-2.1 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cairo compizconfig dbus fuse gsettings +gtk gtk3 inotify +svg marco mate"
+REQUIRED_USE="compizconfig? ( gtk )
+	gsettings? ( gtk )
+	marco? ( gsettings )" #338071
+
+COMMONDEPEND="
+	>=dev-libs/glib-2
+	dev-libs/libxml2
+	dev-libs/libxslt
+	media-libs/libpng:0=
+	media-libs/mesa
+	x11-base/xorg-server
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	x11-libs/libICE
+	x11-libs/libSM
+	>=x11-libs/libXrender-0.9.3
+	>=x11-libs/startup-notification-0.7
+	virtual/glu
+	x11-libs/pango
+	cairo? ( >=x11-libs/cairo-1.4[X] )
+	compizconfig? (
+		>=x11-libs/libcompizconfig-0.8
```

Comment:
You need to order the commits so that the dependencies come first, i.e. that every commit is green by `repoman full -d` (and yes, you're supposed to test that).